app/res/xml/preferences_developer.xml

```diff
@@ -45,6 +43,11 @@
         android:entryValues="@array/pref_enabled_vals"
         android:key="cc-enable-session-saving"
         android:title="Enable Session Saving"/>
+    <EditTextPreference
```

Comment:
What do you think about having this only be visible if session saving is enabled?